Cargo.toml

```diff
@@ -24,3 +28,4 @@ tempfile = "3.1.0"
 lazy_static = "1.3.0"
 serde_json = "1.0.40"
 serde = { version = "1.0.98", features = ["derive"] }
+test-case = "1.1.0"
```

Comment:
I cannot find any usage of `test-case` in this PR.

Reply:
Opps, thanks for spotting that.